eurostag-step-up-transformer/pom.xml

```diff
@@ -77,5 +77,10 @@
             <artifactId>slf4j-simple</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.jimfs</groupId>
+            <artifactId>jimfs</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
The dependency should be sorted by scope (compile, runtime, test), by groupId and artifactId.